crates/bevy_render/src/view/mod.rs

```diff
@@ -282,14 +282,16 @@ pub struct ExtractedView {
     /// cw = -right - left
     /// ch = -top - bottom
     ///
-    /// ⎡  2 / w       0         0  0 ⎤
-    /// ⎢      0   2 / h         0  0 ⎥
-    /// ⎢      0       0     1 / d  0 ⎥
-    /// ⎣ cw / w  ch / h  near / d  1 ⎦
+    /// ⎡ 2 / w      0      0    cw / w ⎤
+    /// ⎢     0  2 / h      0    ch / h ⎥
+    /// ⎢     0      0  1 / d  near / d ⎥
```

Comment:
I don't know this codebase, but a quick check: is this row correct given the swapping of near and far here https://github.com/bevyengine/bevy/blob/main/crates/bevy_render/src/camera/projection.rs#L561-L562 ?

Reply:
good catch, thanks!